infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -118,12 +121,21 @@
      *
      * {@link SequencerServer::maxConflictWildcard} :
      * a "wildcard" representing the maximal update timestamp of
-     * all the confict keys which were evicted from the cache
+     * all the conflict keys which were evicted from the cache
      */
     private long maxConflictWildcard = Address.NOT_FOUND;
```

Comment:
Looks like this variable should be volatile or AtomicLong

Reply:
@xnull why? it is only accessed in synchronized methods. @annym will there be unlocked access/write to this?

Reply:
I was talking about anny's "maxConflictNewSequencer" being accessed in synchronized method. This one however might and needs investigation.

Reply:
Ok, but it seems dangerous anyway

Reply:
Just realized - we read maxConflictNewSequencer in txCanCommit method which is not synchronized, so if I am not wrong - it's possible we could update maxConflictNewSequencer in synchronized method but read stale value in  txCanCommit method.
Am I right?

Reply:
It doesn't really matter. The sequencer server is single threaded anyways.

Reply:
This fact is specific only for current SequenceServer, what if tomorrow we decide to replace `newSingleThreadExecutor` with another one?

Reply:
I think that will be a bigger change, not as seemless as just swapping the thread executor.